### PR TITLE
feat: Official notebook for Mistral AI models in Model Garden [Launch…

### PR DESCRIPTION
**Pull request by sujituk**
…: 07/24]

**REQUIRED:** Add a summary of your PR here, typically including why the change is needed and what was changed. Include any design alternatives for discussion purposes.

<br>
Added the official colab notebook to get started with Mistral AI's 3 models
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [x] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [x] Follow the style and grammar rules outlined in the above notebook template.
- [x] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [x] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [x] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [x] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/generative_ai/mistralai_intro.ipynb [hunk 1]
```diff
@@ -0,0 +1,332 @@
+{
```
Comment (gericdong) [inline notebooks/official/generative_ai/mistralai_intro.ipynb:1]: <p>Just checking if this statement is intended: "All Mistral AI models can to be updated"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/3283/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Comment (sujituk) [reply notebooks/official/generative_ai/mistralai_intro.ipynb:1]: Fixed issue

File: notebooks/official/generative_ai/mistralai_intro.ipynb [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+        "id": "8fK_rdvvx1iZ"
+      },
+      "source": [
+        "## Overview\n",
+        "\n",
+        "### Mistral AI on Vertex AI\n",
+        "\n",
+        "Mistral AI models on Vertex AI offer fully managed and serverless models are offered as managed APIs. To use a Mistral AI model on Vertex AI, send a request directly to the Vertex AI API endpoint.\n",
+        "\n",
+        "You can stream your MIstral AI model responses to reduce the end-user latency perception. A streamed response uses server-sent events (SSE) to incrementally stream the response.\n",
```
Comment (kittyabs) [inline notebooks/official/generative_ai/mistralai_intro.ipynb:71]: Is this a typo? "MIstral AI model" - upper case "I"

**Review by kittyabs — COMMENTED** [review]
typo

File: notebooks/official/generative_ai/mistralai_intro.ipynb (hunk 2 above)
Comment (sujituk) [reply notebooks/official/generative_ai/mistralai_intro.ipynb]: typoe fixed: https://github.com/GoogleCloudPlatform/vertex-ai-samples/pull/3283/commits/a492c4264a243559e51ad304843f0681c9b8493e

**Comment by sujituk** [comment]
Merge conflicts. Will close PR and submit a new one.

**Pull request closed without merging**